modules/web/client/src/main/scala/observe/ui/components/queue/SessionQueue.scala

```diff
@@ -195,10 +196,10 @@ object SessionQueue:
     ),
     ColDef(
       ObsRefColumnId,
-      _.obsReference,
+      obs => obs.obsReference.toRight((obs.programId, obs.obsId)),
```

Comment:
Right, so we use the reference if available and default to progid/obsid if not
A question is whether we should load obs without a reference, I guess it will be needed for engineering?